department-of-veterans-affairs/va.gov-cms
# VACMS-10739: Cache preventing options from showing

## Description

Closes #10739.

## Testing done
Manually.

## Screenshots
<img width="1840" alt="Screen Shot 2022-11-03 at 12 04 39 AM" src="https://user-images.githubusercontent.com/766573/199652299-86220fba-de02-4c62-87af-70e9da4b1691.png">


## QA steps
Because the fix to this bug is to create more specific caches, the only way to test it (especially with memcache running), is to simply edit content types as the user. Even though we never determined a replicable set of steps to trigger the error, a tester should still no longer see the following on the VAMC Facility Detail Page:
![Error](https://user-images.githubusercontent.com/9059428/190473645-c403c7ff-a06b-481b-85aa-2cc8d1b63e62.png)


As stefanie.gray, edit a [Vet Center Facility Service](https://pr11412-50kwcdiqgmlwz7pgfz485okslscdo6mn.ci.cms.va.gov/node/41100/edit).
- [x] Validate that you can successfully edit the page.
- [x] _Note: there's no need to save it_

As stefanie.gray, start the process to create a new [Vet Center Service](https://pr11412-50kwcdiqgmlwz7pgfz485okslscdo6mn.ci.cms.va.gov/node/add/vet_center_facility_health_servi).
- [x] Validate that you select from the following dropdowns:
  - [x] Section
  - [x] Facility name
- [x] _Note: there's no need to save it_

As stefanie.gray, edit a [VAMC Detail Page](https://pr11412-50kwcdiqgmlwz7pgfz485okslscdo6mn.ci.cms.va.gov/node/35189/edit).
- [x] Validate that the "Related Office or Health Care System" dropdown has "VA Milwaukee Health Care" selected
- [x] Try to edit a Rich Text paragraph under "Main Content"
- [x] Validate that it throws no error
- [x] _Note: there's no need to save it_

As stefanie.gray, start the process to create a new [VAMC Detail Page](https://pr11412-50kwcdiqgmlwz7pgfz485okslscdo6mn.ci.cms.va.gov/node/add/health_care_region_detail_page)
- [x] Select a "Section" of your choosing
- [x] Select a "Related Office or Health Care System" of your choosing
- [x] Validate that the "Related Office or Health Care System" field is populated
- [x] _Note: there's no need to save it_

### Definition of Done


### Select Team for PR review





### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
